src/components/Modal/Modal.vue

```diff
@@ -498,15 +498,23 @@ $header-size: 50px;
 	.modal-title {
 		overflow-x: hidden;
 		box-sizing: border-box;
-		max-width: 100%;
-		padding: 0 #{$clickable-area * 2}; // maximum actions is 2
+		width: 100%;
+		padding: 0 #{$clickable-area * 3} 0 12px; // maximum actions is 2 but we violate it here
```

Comment:
what is this 12 px?

Reply:
"but we violate it here" also, what is this? :p

Reply:
> what is this 12 px?

Just a padding we often use, and based on a scale of 4, which was proposed by Jenni. Yes it needs standardization. Something for a separate pull request though as it requires more work, but this is an urgent fix.



> "but we violate it here" also, what is this? :p

Just to clarify that normally it should be 2 actions max. ;) And it literally had the *2 in there, but was showing 3 icons.
In this case actually 3 is fine though cause the close X is extra.

Reply:
> Just a padding we often use, and based on a scale of 4, which was proposed by Jenni. Yes it needs standardization. Something for a separate pull request though as it requires more work, but this is an urgent fix.

Then please use $icon-margin as this si the one we use in viewer.
We never use random px values here :)

> And it literally had the *2 in there, but was showing 3 icons.

Yeah, because the close icon is not in the viewer.
Please put back the original comment as this one doesn't help :p

Reply:
> Yeah, because the close icon is not in the viewer.

actually, wrong answer, this is because the slideshow is not an `Actions`

Reply:
> Then please use $icon-margin as this is the one we use in viewer.

I was looking at icon-margin, but this has nothing to do with icons?

Also, regarding the random px values, see https://github.com/nextcloud/nextcloud-vue/pull/764/files#diff-a25f5501f9ebbd73f59b3335f9545c4bR530 ;)

Reply:
@skjnldsv please add a commit with the changes you want, or propose them as suggestions in commits. It’s difficult to keep track otherwise.

Reply:
> Also, regarding the random px values, see [nextcloud/nextcloud-vue/pull/764/files#diff-a25f5501f9ebbd73f59b3335f9545c4bR530](https://github.com/nextcloud/nextcloud-vue/pull/764/files#diff-a25f5501f9ebbd73f59b3335f9545c4bR530) ;)

AAArgh :grin: 
/me shivers

Reply:
> I was looking at icon-margin, but this has nothing to do with icons?

it's indexed on the clickable area :)